classes/admin.php

```diff
@@ -747,6 +747,12 @@ public function getPage($path)
 
                 $header = ['title' => $data['title']];
 
+                // Merge the default page headers into $header
+                $default_headers = Yaml::parse(
+                  $this->grav['config']->get('plugins.admin.frontmatter.default_headers'));
+
+                $header = (object) array_merge((array) $header, (array) $default_headers);
+
```

Comment:
Funny, a request came up today on twitter for a plugin that is exactly the kind of thing that would benefit from this event, so I just added it and created the simple plugin.

Here's the commit that adds the event: https://github.com/getgrav/grav-plugin-admin/commit/1ea0a5490319e91609dc1b5e04243832595b3b2e

Auto Date Plugin: https://github.com/getgrav/grav-plugin-auto-date

You can use this plugin as an example to create your own.  Frankly it took me 10 minutes with the help of `devtools` plugin to create the basic plugin - https://github.com/getgrav/grav-plugin-devtools